saferpayofficial.php

```diff
@@ -21,6 +21,8 @@
  *@license   SIX Payment Services
  */
 
+use Invertus\SaferPay\Repository\SaferPayOrderRepository;
```

Comment:
If module is supported in 1.6, then do not use dependency import here. If you need to define service namespace, use full namespace where you need that (probably other services are use there like that as well).